config/installDependencies.py

```diff
@@ -75,3 +83,12 @@
 			shutil.rmtree( f )
 		else :
 			os.remove( f )
+
+# Windows images don't include Doxygen
+if platform == "windows" :
+	doxygenURL = "http://doxygen.nl/files/doxygen-1.8.15.windows.x64.bin.zip"
+
+	sys.stderr.write( "Downloading Doxygen \"%s\"" % downloadURL )
+	doxygenFileName, headers = urllib.urlretrieve( doxygenURL )
+
+	os.system( "7z x %s -o%s\\doxygen -aoa -y" % ( doxygenFileName, dependenciesDir ) )
```

Comment:
As per earlier note, can we keep this to just installing the compile deps, and put this in an `Install Toolchain (Windows)` task along with the `scons` install?

Reply:
Taken care of in fe1733e